src/components/number_input_wrapper.tsx

```diff
@@ -23,7 +25,7 @@ interface NumberInputWrapperProps {
  * represents a number of dollars)
  * @param suffix Text to appear as a suffix to the NumberInput (e.g. "SqFt")
  */
-const NumberInputWrapper: FC<NumberInputWrapperProps> = ({label, path, prefix, suffix}) => {
+const NumberInputWrapper: FC<NumberInputWrapperProps> = ({label, path, prefix, suffix, min=0, max=100000000}) => {
```

Comment:
Set defaults for min and max: min=Number.NEGATIVE_INFINITY, max=Number.INFINITY.